master/manifests/rbac-etcd-calico.yaml

```diff
@@ -0,0 +1,4 @@
+---
+layout: null
+---
+{% include {{page.version}}/manifests/rbac-etcd-calico.yaml %}
```

Comment:
What's with these new RBAC files (I must be missing something!)?

Reply:
These are there primarily to handle the integration use case. I think it is just a matter of adding the flags, as per @caseydavenport 's new changes. I can take a stab at it.